src/Core/Middlewares/HandleArpiteRequests.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Arpite\Core\Middlewares;
+
+use Illuminate\Foundation\Auth\User;
+use Illuminate\Http\Request;
+use Inertia\Middleware;
+
+class HandleArpiteRequests extends Middleware
+{
+	/** @var string */
+	protected $rootView = "arpite::application";
+
+	/**
+	 * @param Request $request
+	 * @return array<int, mixed>
+	 */
+	public function share(Request $request): array
+	{
+		/** @var User|null $user */
+		$user = auth()->check() ? auth()->user() : null;
+
+		return array_merge(parent::share($request), [
+			"user" => $user?->only("email", "name"),
+		]);
```

Comment:
Couldn't we also move all other values from AppServiceProvider `boot` method (except `balance`)?